controllers/datafoundationclient.go

```diff
@@ -122,9 +136,30 @@ func (r *dataFoundationClientReconciler) parseSpec(offering *v1alpha1.ManagedFus
 }
 
 func (r *dataFoundationClientReconciler) reconcilePhases() (ctrl.Result, error) {
+	// Checking for CRDs that are installed after the agent was started and add watchers for the new CRDs
```

Comment:
We are not adding any watches right now.
```suggestion
	// Checking for CRDs that were installed after the agent was started
```

Reply:
done